iaso/api/form_attachments.py

```diff
@@ -41,16 +43,19 @@ def validate(self, data: typing.MutableMapping):
 
     def create(self, validated_data):
         form: Form = validated_data["form"]
-        file: File = validated_data["file"]
+        file: InMemoryUploadedFile = validated_data["file"]
+
+        is_safe, details = scan_uploaded_file_for_virus(file)
+        if not is_safe:
+            raise exceptions.ValidationError({"file": [details]})
```

Comment:
from the screenshot and my test this return a 500, I would add a test on that endpoint
to keep beanstalk "healthy" I would return a 409 or something more "client" related.

I would also add a test on that endpoint with a mocked clamav 

and I don't know if it's a dev env behaviour but I get this
instead of just showing the red box below.

![image](https://github.com/user-attachments/assets/106367a3-7788-43fb-a890-952d2cc4086d)

Currently the message is not super related to a "virus scan"
```
{
  "name": "ApiError",
  "status": 500,
  "message": "Error on POST /api/formattachments/"
}
```
while the logs says
```
django.core.exceptions.ValidationError: {'file': ['A virus was found in this file, please contact your administrator']}
``

Reply:
400 is now returned

@beygorghor how to avoid the "uncought" runtime errors ?